rfd/0151-database-permission-management.md

```diff
@@ -78,6 +117,7 @@ spec:
       add_labels:
         env: prod
         product: WidgetMaster3000
+        local_id: '{{obj.object_kind}}-{{obj.schema}}-{{obj.name}}'
```

Comment:
Is a multi-var template really needed? For labels matching, it feels easier to me to match multiple single-purpose labels than matching a big label like `local-id`.

Reply:
I felt this is more natural given how often database objects are referenced by their fully qualified name, but I'm happy to trim that complexity from the PR if you think this isn't needed. We can always add it later, as an extension: `'{{obj.object_kind}}-{{obj.schema}}-{{obj.name}}'` would be invalid otherwise, so adding support would mean making valid previously invalid configurations.

Reply:
```yaml
match:
  local_id: 'table-sales-cars'
```
vs 
```yaml
match:
  kind: table
  schema_name: sales
  object_name: cars
```
I still think the latter is easier and more flexible.

That said, I am not super against having the compound var support here. You can keep it if you like. I just think it is not necessary, and the implementation would be easier without it.

Reply:
I've dropped multi-interpolation, although we can easily get it back if needed.